### PR TITLE
some refactoring

### PR DESCRIPTION
**Pull request by andreas-may**

### CODE REVIEW

File: src/db.js [hunk 1]
```diff
@@ -358,24 +357,34 @@ function findAreaIdForLocation(location) {
             .then(getMatchingArea);
 }
 
+function close() {
+  db.close();
+}
+
+function remove(filename) {
```
Comment (freethan) [inline src/db.js]: Is this used ?

File: src/db.js [hunk 2]
```diff
@@ -358,24 +357,34 @@ function findAreaIdForLocation(location) {
             .then(getMatchingArea);
 }
 
+function close() {
+  db.close();
```
Comment (freethan) [inline src/db.js]: Maybe an overkill, but consider also

```
function close() {
  db.close(function (err) {
    if (err) {
      throw err;
    }
    console.log("Closing DB!");
  });
}
```

Comment (freethan) [inline src/db.js]: Besides the above, looks ok to me 👍

File: src/db.js (hunk 1 above)
Comment (andreas-may) [inline src/db.js]: Yes, this is used by the test suite added in the add-tests branch.

**Pull request merged**